domain/src/main/java/com/teamagam/gimelgimel/domain/geometries/GeometryInteractor.java

```diff
@@ -0,0 +1,60 @@
+package com.teamagam.gimelgimel.domain.geometries;
+
+import com.teamagam.gimelgimel.domain.base.executor.PostExecutionThread;
+import com.teamagam.gimelgimel.domain.base.executor.ThreadExecutor;
+import com.teamagam.gimelgimel.domain.base.interactors.AbstractInteractor;
+import com.teamagam.gimelgimel.domain.geometries.entities.VectorLayer;
+import com.teamagam.gimelgimel.domain.geometries.repository.GeoRepository;
+import com.teamagam.gimelgimel.domain.messages.entity.MessageGeo;
+import com.teamagam.gimelgimel.domain.messages.repository.MessagesRepository;
+
+import rx.Observable;
+import rx.Subscriber;
+
+public class GeometryInteractor extends AbstractInteractor {
+
+    private static final String LAYER_ID = "UserLayer";
```

Comment:
I'm not sure this is the right place for this string.
also, this is not "UserLayer", but "SentMessagesLayer"
(same for LAYER_ID)